src/App.vue

```diff
@@ -93,6 +94,7 @@ import { updateLdUser } from '@/utils'
 import PaySystemAlert from 'sbc-common-components/src/components/PaySystemAlert.vue'
 import SbcHeader from 'sbc-common-components/src/components/SbcHeader.vue'
 import SbcFooter from 'sbc-common-components/src/components/SbcFooter.vue'
+import { BreadCrumb } from '@bcrs-shared-components/bread-crumb'
```

Comment:
Not a big deal, but see if you can move this import to src/components/common/index.ts and then, here, import from there. (Purpose is to remove external imports from our components.)

Reply:
100% i was looking at doing that but there was no default index export for the components so i didn't want to break the mould per se of Filings UI. 
Can definitely add that in!

Reply:
That's why I thought "common" might be OK.

Reply:
Definitely, good call. Will do!

Reply:
Done in next commit